pkg/3scale/amp/component/apicast_monitoring.go

```diff
@@ -151,6 +151,18 @@ func ApicastPrometheusRules(ns string) *monitoringv1.PrometheusRule {
 								"severity": "warning",
 							},
 						},
+						{
+							Alert: "ThreescaleApicastWorkerRestart",
+							Annotations: map[string]string{
+								"summary":     "A worker process in Nginx has been started",
```

Comment:
What about adding the word `new` into the summary, to let us know that a new worker process has been started? I am presuming that this alert won't fire on creation of the initial worker process, but only if a new one if created (I could be wrong here.). I.e `A new worker process in Nginx has been started`? WDYT?

Reply:
+1

Reply:
Changed.